app/controllers/goldencobra/sessions_controller.rb

```diff
@@ -54,5 +50,14 @@ def logout
 
     def register
     end
+
+    private
+
+    def usermodel
+      return false if params[:usermodel].blank?
+      return false unless %w(user visitor).include?(params[:usermodel].to_s.downcase)
```

Comment:
Könnte die Whitelist dann nicht eine Konstante sein `X = %w(user visitor)`, die wahlweise in einzelnen Projekten überschrieben werden kann, wenn nötig?